packages/dcos-integration-test/extra/test_tweeter_demo.py

```diff
@@ -0,0 +1,86 @@
+import os
+import time
+
+import requests
+
+from test_packaging import (
+    _skipif_insufficient_resources
+)
+
+DEMO_RESOURCE_REQUIREMENTS = {
+    'number_of_nodes': 5,
+    'node': {
+        'disk': 0,
+        'mem': 0,
+        'cpus': 1
+    }
+}
+
+
+def _get_tweeter_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/tweeter.json')
+    return response.json()
+
+
+def _get_post_tweets_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/post-tweets.json')
+    return response.json()
+
+
+def _get_tweet_count(address):
+    """Count the number of tweets on the tweeter app's webpage
+
+    Args:
+        address: str URL including http://
+    """
+    response = requests.get(address)
+    return response.text.count('tweet-content')
+
+
+def test_tweeter_demo_oss(dcos_api_session):
```

Comment:
This is only being skipped implicitly downstream because there are not enough resources to run this test. Rather, can we add a `pytest.skipif(test_helpers.expanded_config.get('security') == 'strict', reason='This test requires specific service accounts if security is strict'`